src/auditor/checks.py

```diff
@@ -298,11 +298,36 @@ def title_check(self):
         #: Always include the old title for readability
         title_data = {'title_fix': 'N', 'title_old': self.item.title, 'title_new': ''}
 
-        if self.new_title and self.new_title != self.item.title:
-            title_data = {'title_fix': 'Y', 'title_old': self.item.title, 'title_new': self.new_title}
+        new_title = None
 
+        #: If we have a new title from metatable, add and check for deprecated note
+        if self.title_from_metatable and self.title_from_metatable != self.item.title:
+            new_title = self.title_from_metatable
+            if self.authoritative == 'deprecated' and 'deprecated' not in new_title.casefold():
+                new_title += ' (Deprecated)'
```

Comment:
Hmm, good question... prepending would make it more visible, but would also raise them to the top of any alphabetical sort... not sure if there are any places where they're sorted alphabetically... @gregbunce, any thoughts here?

I'm also not sure if changing the title would change how it's displayed/named in any existing maps/webmaps using that layer, so I'm not sure how beneficial this would be if that's the case.

Reply:
I tested the title change and they do not update in web maps or pro projects so you can disregard these suggestions. I might still suggest prepending something that would sort these items last though.

I assumed from reading https://odoe.net/blog/arcgis-deprecated-items-in-jsapi

> Whatever the reason, when the item is marked as deprecated, and you are using it in an application, it would be cool to figure that out. If you're lucky, they added deprecated to the title. Sometimes, they don't.

But that isn't the case.

Reply:
Yeah, I can see both sides of that coin- it'd be great to be able to notify current users of those layers, but I'd get frustrated if I renamed a layer in a webmap and then suddenly it changed on me.